src/main/kotlin/org/kamiblue/client/module/modules/movement/Strafe.kt

```diff
@@ -57,6 +57,7 @@ internal object Strafe : Module(
     private val strafeTimer = TickTimer(TimeUnit.TICKS)
 
     /* If you skid this you omega gay */
+    /* ^ ez ban for sexual harassment I'm not gay tho*/
```

Comment:
To be quite honest having the original comment was unprofessional enough, I'd rather a second not be added. Just remove both if you'd like in this pr, or just yours.